### DISCUSSION TITLE
New Riverscapes Context Fixes in RAVE!

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#203] [Show and tell]
Hello @Riverscapes/riverscape-project-curators,
Just wanted to say thanks to the entire team of curators for doing such a great job pushing things along and @MattReimer for helping us all. Last night, @MattReimer merged a big pull request for QRAVE symbology for the RS_Context (#191) that @Tanmanfalcon had been cranking on and #193 on BRAT that @tyler1218hatch had been working on. As two of our most regularly used projects, you'll notice some big changes/improvements in those from QRAVE (note changes are being made to ArcRAVE but they won't be visible until [next release of ArcRAVE](https://github.com/Riverscapes/RaveAddIn/issues/137)).

Anyhow, have a test drive, and if you have specific comments, suggestions or requests, post them. Nice job folks!

Cheers,
Joe

**Comment by wally-mac** [comment]
I'd like to underscore the thanks that @joewheaton extended to @MattReimer ,  @Cashe93 , and @Tanmanfalcon . You guys have really stepped up to the plate and you're knocking things out of the park. Here's to more curation successes!
